src/libs/OptionsListUtils.js

```diff
@@ -1043,6 +1054,167 @@ function getTagListSections(rawTags, recentlyUsedTags, selectedOptions, searchIn
     return tagSections;
 }
 
+/**
+ * Represents the original tax rates object.
+ *
+ * @typedef {Object} PolicyTaxRates
+ * @property {Object.<string, TaxData>} taxes - Object containing tax rates with their codes.
+ */
+
+/**
+ * Represents the data for a single tax rate.
+ *
+ * @typedef {Object} TaxData
+ * @property {string} name - The name of the tax rate.
+ * @property {string} value - The value of the tax rate.
+ * @property {string} code - The code associated with the tax rate.
+ * @property {string} actualName - The actual name of the tax rate.
```

Comment:
Why actualName here, What does it hold?

Reply:
I have updated this to `modifiedName` which is a new field holding the name and the value in this form `tax Rate 2 (5%)`.

Reply:
modifiedName is much better, thanks!